lib/Util/RequestOptions.php

```diff
@@ -21,6 +21,9 @@ class RequestOptions
     /** @var null|string */
     public $apiBase;
 
+    /** @var null|string */
+    public $connectBase;
```

Comment:
I'm a little bit on the fence about this. `RequestOptions` contains options that affects a specific request. A specific request goes to a specific endpoint. IMO there should be a single parameter that contains the base URL for the request.

It's what we did in stripe-dotnet (where it is more appropriately called [`BaseUrl`](https://github.com/stripe/stripe-dotnet/blob/master/src/Stripe.net/Services/_common/RequestOptions.cs#L26-L32)), and what I had in mind when I designed the PHP services, though the only existing use case is file creation requests: https://github.com/stripe/stripe-php/blob/master/lib/Service/FileService.php#L53-L56.